install.sh

```diff
@@ -391,7 +396,7 @@ dockerPerformSettings='./config/dev_settings.txt'
 
 scriptName=$0
 skipswarm=0
-defUser=$(who | awk '{print $1;}')
+defUser=$(who | awk 'NR==1{print $1;}')
```

Comment:
shouldn't this be just ?

defUser=$(whoami)

Reply:
Or perhaps use `id -un` if whoami is problematic in some environments

My concern is with the output of `who`. It returns all the users logged in (ordered by log-in time?) which could cause defUser being set to a different username.

We should also investigate the possibility of not requiring sudo as part of installation and rather require that the user can change docker configuration as a pre-requisite